res_country_statistics/models/models.py

```diff
@@ -0,0 +1,90 @@
+##############################################################################
+#
+#    Copyright (C) 2020 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: David Wulliamoz <dwulliamoz@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import api, models, fields
+from pandas_datareader import wb
+from datetime import date
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class ResCountryInfo(models.Model):
+    """ add latest stat to the country """
+    _inherit = "res.country"
+    capital_city = fields.Char("capital city")
+    statistics_ids = fields.Many2many(
+        "res.country.stat",
+        "res_country_stat_latest",
+        string="latest stat",
+        compute="_compute_latest_stats",
+        track_visibility="onchange",
+    )
+
+    @api.multi
+    def _compute_latest_stats(self):
+        indicators = self.env['res.country.indicator'].search([])
+        for record in self:
+            stat = self.env['res.country.stat'].search([('country_id', '=', record.id)])
+            country_statistics = []
+            for i in indicators:
+                res = stat.search([('indicator_id', '=', i.id)], order='year desc', limit=1)
+                if res:
+                    country_statistics.append(res.id)
+            self.statistics_ids = country_statistics
```

Comment:
Better like this (more understandable + bug fix in assignation):
```suggestion
        indicators = self.env["res.country.indicator"].search([])
        for country in self:
            country_stats = self.env["res.country.stat"]
            for i in indicators:
                country_stats += self.env["res.country.stat"].search([
                    ("country_id", "=", record.id),
                    ("indicator_id", "=", i.id)
                ], order="year desc", limit=1)
            country.statistics_ids = country_stats
```

Reply:
What do you think of this change?